examples/crm-full-stack/stripe-app/stripe-app.json

```diff
@@ -0,0 +1,29 @@
+{
+  "id": "com.saasbase.stripe-app-fe",
+  "version": "0.0.1",
+  "name": "CRM Buddy",
+  "icon": "",
+  "permissions": [],
```

Comment:
Although not required in your sample,  a full stack end app should make API requests to Stripe as well for xyz data (e.g. Customer object). This would showcase the use case utilizing stripe data. I'd suggest adding in small request and handling that data via your backend. If you do so, you'll need the respective permission listed here.

Reply:
Yep absolutely. An example of working with Stripe objects is indeed needed. I have another PR in the works for precisely this. 

As for this PR, the aim is to show how a Stripe App frontend can be connected to a backend API service (and DB) using a very real example use case - Adding notes to Customers.